services/httpd/privatebin.nix

```diff
@@ -0,0 +1,20 @@
+{ pkgs, ... }:
+with (import ./shared.nix);
+let
+  user = "wiki";
+  group = "redbrick";
+in {
+  services.httpd.virtualHosts = [
+    (vhost {
+      user = "${user}";
```

Comment:
do  we need to wrap this in `"${}"`

Reply:
No it could just be user, or even better for these 2 lines `inherit user group;`

Reply:
I figured that was how to substitute values - what should it be instead?